test/spec/runners/epoch_processing.ex

```diff
@@ -38,24 +40,34 @@ defmodule EpochProcessingTestRunner do
 
     config = SpecTestUtils.get_config(testcase.config)
 
-    _pre =
+    pre =
       SpecTestUtils.read_ssz_from_file!(
         case_dir <> "/pre.ssz_snappy",
         SszTypes.BeaconState,
         config
       )
 
-    {:ok, _post} =
+    {:ok, post} =
       SpecTestUtils.read_ssz_from_file(
         case_dir <> "/post.ssz_snappy",
         SszTypes.BeaconState,
         config
       )
 
-    # handle_case(testcase.handler, pre, post)
+    handle_case(testcase.handler, pre, post)
   end
 
   # def handle_case("eth1_data_reset", pre, post) do
   #   assert process_eth1_data_reset(process_eth1pre.eth1_data) == post.eth1_data
   # end
+
+  def handle_case("effective_balance_updates", pre, post) do
+    result = EpochProcessing.process_effective_balance_updates(pre)
+
+    if is_nil(post) do
+      assert elem(result, 0) == :error
```

Comment:
nit: assert `{:error, _reason } = result`

Reply:
now that we are changing the spec, this condition would never evaluate to true, so should I just skip it?

Reply:
yes, sorry, that comment is outdated not